sections/bulk-quick-order-list.liquid

```diff
@@ -0,0 +1,25 @@
+{{ 'component-price.css' | asset_url | stylesheet_tag }}
+{{ 'quick-order-list.css' | asset_url | stylesheet_tag }}
+{{ 'quantity-popover.css' | asset_url | stylesheet_tag }}
+
+<script src="{{ 'quantity-popover.js' | asset_url }}" defer="defer"></script>
+<script src="{{ 'price-per-item.js' | asset_url }}" defer="defer"></script>
+<script src="{{ 'quick-order-list.js' | asset_url }}" defer="defer"></script>
```

Comment:
Trying to think whether this is all needed. Where is this section potentially loaded ?
- featured collection - here we already load -> `component-price.css`, `quantity-popover.js`, `price-per-item.js`, `quick-order-list.js`
- collection template - here we already load -> `component-price.css`, `quantity-popover.js`, `price-per-item.js`, `quick-order-list.js`
- quick order list section - here we already load -> `component-price.css`, `quick-order-list.css`, `quantity-popover.css`, `quantity-popover.js`, `price-per-item.js`, `quick-order-list.js`

So I think there might be an opportunity to remove some asset calls from here. All the JS files. As for the CSS assets, we could potentially call those assets (`quick-order-list.css` and `quantity-popover.css`) in `featured-collection.liquid` file and `main-collection-product-grid.liquid` within the existing conditional: 
```liquid
{%- if section.settings.quick_add == 'bulk' -%}
```

🤷 At least seems like the JS ones could be removed. Unless I'm missing a place where bulk quick order is present.